### ISSUE TITLE
Roadmap to Kerkerkruip 10

### ISSUE THREAD

**Issue by curiousdannii** [i7/kerkerkruip#303]
This issue is a place for us to discuss Kerkerkruip 10: what we want to include in it, the process of getting a release ready etc.

**Comment by mciul** [comment]
I think we're getting close! @VictorGijsbers, you sent an email out a couple months ago describing the situation - could you summarize what still applies from that here?

**Comment by VictorGijsbers** [comment]
I guess this can be closed.
